actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -595,16 +595,20 @@ def new_instance_with_inherited_permitted_status(hash)
         end
       end
 
-      def convert_parameters_to_hashes(value)
+      def convert_parameters_to_hashes(value, allow_unsafe:)
```

Comment:
Would probably change to a regular arg called `using` that was just the method to send.